k9mail-library/src/test/java/com/fsck/k9/mail/filter/SmtpDataStuffingTest.java

```diff
@@ -0,0 +1,79 @@
+package com.fsck.k9.mail.filter;
+
+import java.io.IOException;
+import java.io.ByteArrayOutputStream;
+
+import okio.ByteString;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.Assert.assertEquals;
+
+public class SmtpDataStuffingTest {
+    private String expectedMessageAfterStuffing;
+    private String actualMessageAfterStuffing;
+
+    @Before
+    public void setUp(){
```

Comment:
Sure, I'll add a space (of course, if I move the fields inside the methods this wouldn't be needed).